app/models/classification.rb

```diff
@@ -36,7 +37,16 @@ def self.scope_for(action, user, opts={})
     when :incomplete
       incomplete_for_user(user).includes(:subjects)
     when :project
-      joins(:project).merge(Project.scope_for(:update, user)).includes(:subjects)
+      return none if opts[:last_id] && !opts[:project_id]
+      if opts[:project_id]
+        project_scope = joins(:project)
+                        .includes(:subjects)
+                        .merge(Project.scope_for(:update, user))
+        project_scope = project_scope.after_id(opts[:last_id]) if opts[:last_id]
```

Comment:
seems we can just have append the last_id to the main project scope if the param is set instead of the if branch.
```
project_scope = joins(:project).merge(Project.scope_for(:update, user)).includes(:subjects)
project_scope = project_scope.after_id(opts[:last_id]) if opts[:last_id]
project_scope
```